module-types/Cisco/DS-C48S-300AC.yml

```diff
@@ -0,0 +1,12 @@
+---
+manufacturer: Cisco
+model: DS-C48S-300AC
+part_number: DS-C48S-300AC
+weight: 2.42
+weight_unit: lb
+comments: '[Cisco MDS 9148S 16G Multilayer Fabric Switch Data Sheet](https://www.cisco.com/c/en/us/products/collateral/storage-networking/mds-9148s-16g-multilayer-fabric-switch/datasheet-c78-731523.html)'
+power-ports:
+  - name: Input
```

Comment:
Probably best to name this `PSU{module}` so that port name conflicts don't occur